extension/background.js

```diff
@@ -359,4 +368,12 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     });
     return true;
   }
+  if (request.action === 'volume') {
+    somaPlayerBG.setVolume(request.volume);
+    return true;
```

Comment:
I think you still want a `sendResponse` call even if it doesn't pass anything back, like in the 'clear' action above.